### ISSUE TITLE
Agent not learning for maze environment with CRAR

### ISSUE THREAD

**Issue by parthchadha** [VinF/deer#78]
I am trying to reproduce the results of the CRAR agent in the maze environment and am observing that the agent's test reward is not improving at all. It stays at about -5 for all the 250 epochs. Can you please point me to the experiment settings that can reproduce the results?

**Comment by VinF** [comment]
Hi Parth,

Thanks for reaching out. The value for the learning rate of the representation loss in CRAR is set to a different value for that experiment:
https://github.com/VinF/deer/blob/83bf5338d92393a35b79b8bfe2d1c25f30cd52f8/deer/learning_algos/CRAR_keras.py#L583
It must be set at the value self._lr instead of self._lr/5. Please let me know if you still run into problems.

Apologies for this hard coded value. I will make a fix such that it will be easier for other people to reproduce the results.

**Comment by parthchadha** [comment]
Hi Vincent,
Thanks for pointing this out. I am able to reproduce the results now!
